ding/example/trex.py

```diff
@@ -0,0 +1,59 @@
+import gym
+from ditk import logging
+from tensorboardX import SummaryWriter
+
+from ding.model import DQN
+from ding.policy import DQNPolicy
+from ding.envs import DingEnvWrapper, BaseEnvManagerV2
+from ding.data import DequeBuffer
+from ding.config import compile_config
+from ding.framework import task
+from ding.framework.context import OnlineRLContext
+from ding.framework.middleware import OffPolicyLearner, StepCollector, interaction_evaluator, \
+    eps_greedy_handler, CkptSaver, eps_greedy_masker, sqil_data_pusher, data_pusher
+from ding.utils import set_pkg_seed
+from dizoo.classic_control.cartpole.config.cartpole_trex_dqn_config import main_config, create_config
+from ding.entry import trex_collecting_data
+from ding.reward_model import create_reward_model
+import copy
+import easydict
+import os
```

Comment:
Suggestion: put these import module before import ding to make it look better
(you can simply follow this import order:
some official modules --> torch modules --> ding modules --> relative import )